evap/evaluation/management/commands/anonymize.py

```diff
@@ -76,33 +72,63 @@ def anonymize_data(self):
             self.stdout.write("")
             raise
 
-    def randomize_text_answers(self, lorem_ipsum):
-        for text_answer in TextAnswer.objects.all():
-            text_answer.answer = self.lorem(text_answer.answer, lorem_ipsum)
-            if text_answer.original_answer:
-                text_answer.original_answer = self.lorem(text_answer.original_answer, lorem_ipsum)
-            text_answer.save()
 
-    @staticmethod
-    def lorem(text, lorem_ipsum):
-        word_count = len(text.split(' '))
-        return ' '.join(itertools.islice(itertools.cycle(lorem_ipsum), word_count))
-
-    @staticmethod
-    def generate_random_usernames(first_names, last_names):
-        user_count = UserProfile.objects.count()
-        random_usernames = set()
-        while len(random_usernames) != user_count:
-            random_usernames.add((random.choice(first_names), random.choice(last_names)))
-        return random_usernames
-
-    @staticmethod
-    def replace_usernames_emails(random_usernames):
-        for user, new_username in zip(UserProfile.objects.all(), random_usernames):
+    # Anonymizes the email templates by replacing the actual subject with a sample one
+    # and replacing the body with lorem ipsum.
+    def anonymize_email_templates(self, lorem_ipsum):
```

Comment:
as discussed: please leave this empty for now - the anonymization should be added once we have extended the EmailTemplates by a list of available variables.